testdata/configurations/keyshareserver.yml

```diff
@@ -0,0 +1,25 @@
+verbose: 2
+
+schemes_path: testdata/irma_configuration
+schemes_update: 0
+privkeys: testdata/privatekeys
+url: http://localhost:8080/
+
+keyshare_prefix: /irma_keyshare_server/api/v1/
+keyshare_attribute: test.test.mijnirma.email
+
+db_str: postgresql://testuser:testpassword@postgres:5432/test
```

Comment:
We cannot use `localhost` in the urls due to a limitation in Docker on MacOS. Check the comment in the networks section of `docker-compose.yml`.

Reply:
This means that after this change, this file as well as `myirmaserver.yml` will work only if postgreSQL and Mailhog have been started using Docker compose, and not anymore when they run locally as daemons. Is there no other way that keep both options working?